rawdata/src/main/java/com/griddynamics/jagger/rawdata/receiver/RawDataReceiver.java

```diff
@@ -0,0 +1,103 @@
+package com.griddynamics.jagger.rawdata.receiver;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.griddynamics.jagger.rawdata.protobuf.RawDataPackageProtos;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static com.griddynamics.jagger.rawdata.protobuf.RawDataPackageProtos.RawDataPackage.parseFrom;
+import static com.griddynamics.jagger.rawdata.sender.RawDataSender.KAFKA_KEY_SEPARATOR;
+import static java.util.Collections.singletonList;
+import static org.apache.commons.lang.StringUtils.split;
+
+@SuppressWarnings("unused")
+public class RawDataReceiver {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RawDataReceiver.class);
+    private final String groupId;
+    private final String kafkaTopic;
+    private final Properties kafkaProperties = new Properties();
+    private final Consumer<String, byte[]> consumer;
+
+    /**
+     * @param kafkaTopic   The name of Kafka topic from which data must be received.
+     * @param kafkaServers A list of host/port pairs to use for establishing the initial connection to the Kafka cluster.
+     *                     This list should be in the form "host1:port1,host2:port2,...".
+     * @param groupId      A string that uniquely identifies the group of consumer processes to which this consumer belongs.
+     *                     By setting the same group id multiple processes indicate that they are all part of the same consumer group.
+     */
+    public RawDataReceiver(String kafkaTopic, String kafkaServers, String groupId) {
+        this.kafkaTopic = kafkaTopic;
+        this.groupId = groupId;
+
+        kafkaProperties.put("bootstrap.servers", kafkaServers);
```

Comment:
Might be better to create Consumer/Producer outside Receiver/Sender class and pass it as a dependency.
More testable and SOLID from my point of view.